mindjunkies/forums/tests/test_views.py

```
@@ -0,0 +1,369 @@
+from django.test import TestCase, RequestFactory, Client
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from django.contrib.messages.storage.fallback import FallbackStorage
+from django.http import HttpRequest
+
+from mindjunkies.courses.models import Course, Module
+from mindjunkies.forums.models import ForumTopic, ForumComment, Reply
+from mindjunkies.forums.views import (
+    ForumHomeView, ForumThreadView, ForumThreadDetailsView,
+    TopicSubmissionView, CommentSubmissionView, ReplySubmissionView,
+    LikePostView, LikeCommentView, LikeReplyView
```

Ruff (0.8.2)

`mindjunkies.forums.views.LikePostView` imported but unused

Remove unused import

(F401)